### ISSUE TITLE
The "is-exiting" issue

### ISSUE THREAD

**Issue by buditanrim** [miguel-perez/smoothState.js#190]
Hello. I know some people have been posted this problem,
I've been looking around for 5 days and tried anything, doesn't seem anything works, maybe I'm stupid.

**My condition:**
running on server, not local, I realized running it on local will make it worse.
**First time load page**: Animation success! All good, fadein blablabla.
moving to other page, is-exiting trigger the reverse, which is great (after 3 days figuring it out)
now, the other page loaded, the page blank because is-exiting is there.

my console showing this:

![image](https://cloud.githubusercontent.com/assets/1830184/8445222/2cf9d018-1f68-11e5-956a-3afe9bec794d.png)

Whenever I click to other page, the "is-exiting" always there..

![image](https://cloud.githubusercontent.com/assets/1830184/8445241/5c288136-1f68-11e5-9251-0fa0cf1dd88e.png)

my js is:

![image](https://cloud.githubusercontent.com/assets/1830184/8445307/55faed34-1f69-11e5-851c-595648a0a812.png)

my html:
![image](https://cloud.githubusercontent.com/assets/1830184/8445299/2974f5ca-1f69-11e5-8a66-01a500307825.png)

CSS is the same as the demo page.

can someone help me?

**Comment by shaneparsons** [comment]
You don't have a closing div tag for your `<div id="main" class="m-scene">` which should go after your `<footer></footer>` in this case.

PS. this question would have been better off posted on stackoverflow.

**Comment by buditanrim** [comment]
@shaneparsons Hi, no the html structure above is actually just a dummy and I forget that closing div. However that's not the case

I already solve my issue, just want to share it here
There is few things for anyone to pay attention, such as following:
- the onEnd on the demo is deprecated, change it to onReady/onAfter (this solve one problem)
- don't try it on local development, upload it to server and try it instead.
- somehow put the file before closing <body> is better instead of on the top (head area).

Thanks, I posted on stackoverflow, no one answer it for a week and I see a lot of questions related on stackoverflow been abandoned, so I noticed that and posted it here.

Thanks tho, this is works perfectly for me
